lib/cosmos/api/tlm_api.rb

```diff
@@ -68,7 +71,7 @@ module Api
     def tlm(*args, scope: $cosmos_scope, token: $cosmos_token)
       target_name, packet_name, item_name = tlm_process_args(args, 'tlm', scope: scope)
       authorize(permission: 'tlm', target_name: target_name, packet_name: packet_name, scope: scope, token: token)
-      Store.instance.get_tlm_item(target_name, packet_name, item_name, type: :CONVERTED, scope: scope)
+      CvtModel.get_item(target_name, packet_name, item_name, type: :CONVERTED, scope: scope)
```

Comment:
CvtModel is almost strictly a layer over the Store and doesn't inherit from Model. It almost feels like a Topic to me. Thoughts?